core/org.openjdk.jmc.flightrecorder.rules.jdk/src/main/java/org/openjdk/jmc/flightrecorder/rules/jdk/memory/HeapDumpRule.java

```diff
@@ -0,0 +1,75 @@
+package org.openjdk.jmc.flightrecorder.rules.jdk.memory;
```

Comment:
Must have copyright notice - look for one with Datadog in it to copy. Should have Oracle first, Datadog next, 2020.